justfile

```diff
@@ -67,7 +67,7 @@ docker-build-arm64:
 docker-push-arm64:
     docker push {{ DOCKER_IMAGE }}-arm64
 
-# Publish Docker manifest combining aarch64 and x86 published images
+# Publish Docker manifest combining arch64 and x86 published images
```

Comment:
```suggestion
# Publish Docker manifest combining aarch64 and x86 published images
```

AArch64 is the official name for Arm's 64-bit instruction set architecture (ISA) that was introduced with the Armv8-A update. It almost always refers to [AArch64](https://en.wikipedia.org/wiki/AArch64).

Reply:
ups, my bad 😅